client/landing/stepper/declarative-flow/internals/steps-repository/hosting-verify-email/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { StepContainer } from '@automattic/onboarding';
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { EVERY_FIVE_SECONDS, Interval } from 'calypso/lib/interval';
+import { useSelector } from 'calypso/state';
+import { fetchCurrentUser } from 'calypso/state/current-user/actions';
+import { getCurrentUser } from 'calypso/state/current-user/selectors';
+import VerifyEmail from './verify-email';
+import type { Step } from '../../types';
+import type { UserData } from 'calypso/lib/user/user';
+import './style.scss';
+
+const HostingVerifyEmail: Step = function ImportVerifyEmail( { navigation } ) {
```

Comment:
As per the linked Slack discussion, I think we should move away from calling it "e-mail verification step" and call it "trial description and e-mail verification step". Obviously not named like that 😅 Maybe "HostingTrialDescription" or "Conditions" would be better?